scripts/partition_manager/partition_manager.rst

```diff
@@ -507,7 +509,8 @@ The Partition Manager supports partitions in the external flash memory through t
 Any placeholder partition can specify that it should be stored in the external flash memory region.
 External flash memory regions always use the *start_to_end* placement strategy.
 
-To store partitions in the external flash memory, you must choose a value for the ``nordic,pm-ext-flash`` property in the devicetree.
+To store partitions in the external flash memory, you can choose a value for the ``nordic,pm-ext-flash`` property in the devicetree,
+or directly use external flash DTS node label as DEVICE.
```

Comment:
```suggestion
To store partitions in the external flash memory, you can either choose a value for the ``nordic,pm-ext-flash`` property in the devicetree, or directly use an external flash DTS node label as ``DEVICE``.
```

Is the 2nd part of the sentence supposed to mean that the " external flash DTS node label" should be used as "DEVICE" or that the  external flash DTS node should be labelled as "DEVICE"?

Reply:
The former.